RetrospectiveExtension.Backend/appsettings.json

```diff
@@ -11,5 +11,8 @@
         "Default": "Warning"
       }
     }
+  },
```

Comment:
Adding Default placeholder in appsettings for the backend to make it for local debugging/testing to change this